pulsar-client-cpp/python/pulsar_test.py

```diff
@@ -650,6 +650,110 @@ def _v2_topics(self, url):
 
         client.close()
 
+    def test_topics_consumer(self):
+        client = Client(self.serviceUrl)
+        topic1 = 'persistent://sample/standalone/ns/my-python-topics-consumer-1'
+        topic2 = 'persistent://sample/standalone/ns/my-python-topics-consumer-2'
+        topic3 = 'persistent://sample/standalone/ns/my-python-topics-consumer-3'
+        topics = [topic1, topic2, topic3]
+
+        url1 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-1/partitions'
+        url2 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-2/partitions'
+        url3 = self.adminUrl + '/admin/persistent/sample/standalone/ns/my-python-topics-consumer-3/partitions'
+
+        doHttpPut(url1, '2')
+        doHttpPut(url2, '3')
+        doHttpPut(url3, '4')
```

Comment:
As I mentioned in #2448, I think the multi-topic consumer in C++ is currently not working when there are no partitions in the topic.

Reply:
any actions to address for python after #2448?